src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -251,6 +259,66 @@ export class CourseManagementRequests {
         });
     }
 
+    /**
+     * Creates a submission for a Quiz with only one multiple-choice quiz question
+     * @param quizExercise the response body of a quiz exercise
+     * @param tickOptions a list describing which of the 0..n boxes are to be ticked in the submission
+     */
+    createMultipleChoiceSubmission(quizExercise: any, tickOptions: number[]) {
+        const multipleChoiceSubmission = {
+            ...multipleChoiceSubmissionTemplate,
+            submittedAnswers: [
+                {
+                    ...multipleChoiceSubmissionTemplate.submittedAnswers[0],
+                    quizQuestion: quizExercise.quizQuestions[0],
+                    selectedOptions: tickOptions.map((option) => quizExercise.quizQuestions[0].answerOptions[option]),
+                },
+            ],
+        };
```

Comment:
I think there is too much going on in the spread operator. Pulling the creation of the submittedAnswers out of the operator makes it easier to read in my opinion
```suggestion
        const submittedAnswers = [
                {
                    ...multipleChoiceSubmissionTemplate.submittedAnswers[0],
                    quizQuestion: quizExercise.quizQuestions[0],
                    selectedOptions: tickOptions.map((option) => quizExercise.quizQuestions[0].answerOptions[option]),
                },
            ];
        const multipleChoiceSubmission = {
            ...multipleChoiceSubmissionTemplate,
            submittedAnswers,
        };
```